tests/benchdnn/graph/bench_graph.cpp

```diff
@@ -31,16 +31,18 @@ void check_correctness(const settings_t &s) {
     for_(const auto &i_expected_n_partition : s.expected_n_partition_vec)
     for_(const auto &i_fpmath_mode : s.fpmath_mode_vec)
     for_(const auto &i_dt : s.dt)
+    for_(const auto &i_dts : s.dts)
```

Comment:
what about rename them as i_id_dts and s.id_dts?  we already have ` i_dt ` , i_id_dts is more self-explain than i_dts.

Reply:
Renamed.